eng/code-quality-reports/src/main/resources/checkstyle/checkstyle-suppressions.xml

```diff
@@ -429,6 +429,12 @@ the main ServiceBusClientBuilder. -->
   <suppress checks="IllegalImport" files=".*[/\\]src[/\\]test[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationClientTestBase.java"/>
   <suppress checks="IllegalImport" files=".*[/\\]src[/\\]test[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationTest.java"/>
 
+  <!-- Attestation Client returns AttestationResponse_T_ instead of Response_T_ -->
+  <suppress checks="ServiceClient" files=".*[/\\]src[/\\]main[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationClient.java"/>
+  <suppress checks="ServiceClient" files=".*[/\\]src[/\\]main[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationAsyncClient.java"/>
+  <suppress checks="ServiceClient" files=".*[/\\]src[/\\]main[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationAdministrationClient.java"/>
+  <suppress checks="ServiceClient" files=".*[/\\]src[/\\]main[/\\]java[/\\]com[/\\]azure[/\\]security[/\\]attestation[/\\]AttestationAdministrationAsyncClient.java"/>
+
```

Comment:
This is similar to the issue we have with PagedFlux. We have an issue filed for updating the checkstyle rule.
https://github.com/Azure/azure-sdk-for-java/issues/7499